### DISCUSSION TITLE
Selecting text on the index

### DISCUSSION THREAD

**Discussion by stefblokdijk** [laravel/nova-issues#4276] [Ideas & Feature Requests]
In the previous Nova version it was possible to select a text on the index to quickly copy it. On the current Nova version you go to the resource detail if you try to click on it. It would be nice to make this possible again.

**Comment by philipbaginski** [comment]
Use ->copyable() :-)

**Reply by stefblokdijk** [reply]
This would partly solve the issue, but it seems to be bugged. It doesn't do anything when you click on the text / icon.
